internal/reconciliations/istio/client.go

```diff
@@ -119,6 +121,17 @@ func (c *IstioClient) Uninstall(ctx context.Context) error {
 	return nil
 }
 
+func installIstioInExternalProcess(mergedIstioOperatorPath string) (string, error) {
+	b, err := exec.Command("./istio_install", mergedIstioOperatorPath).Output()
```

Comment:
With this approach we don't see the output of the Istio install while it's running, but only after the process has exited.
We should use the os.Stdout for the cmd, e.g. like this

```
	cmd := exec.Command("./istio_install", mergedIstioOperatorPath)
	cmd.Stdout = os.Stdout
	cmd.Stderr = os.Stderr
	err := cmd.Run()
```